client/app/widgets/Notifications/index.jsx

```diff
@@ -18,117 +18,122 @@ export type State = {
   signedInKey?: number,
 };
 
-export class Notifications extends React.Component<Props, State> {
-  constructor(props: Props) {
-    super(props);
-    this.state = { notifications: '', alreadyMounted: false, open: false };
-  }
+export const Notifications = ({
+  element,
+}: Props) => {
+  const [notifications, setnotifications] = useState('');
+  const [alreadyMounted, setalreadyMounted] = useState(false);
+  const [open, setopen] = useState(false);
+  const [signedInKey, setsignedInKey] = useState(0);
```

Comment:
Since this value can be undefined, we can just initialize it to undefined by doing so `useState()`.
Same for `modalKey` below.

Reply:
Yeah sure